### PR TITLE
feat(uikit): add grid component, add column of different sizes and br…

### PR DESCRIPTION
**Pull request by nearsilen**
…eakpoints

### CODE REVIEW

File: packages/ui/components/Grid.tsx [hunk 1]
```diff
@@ -0,0 +1,155 @@
+import React, { ReactElement } from 'react';
+import cn from 'classnames';
+type Media = 'sm' | 'md' | 'lg' | 'xl' | '2xl' | 'all';
+type MapMedia = {
+  [key in Media]: string;
+};
+interface ContainerProps {
+  children: ReactElement;
```
Comment (feodorrp) [inline packages/ui/components/Grid.tsx:8]: `ReactElement` is not the right type for react children. look at [here](https://blog.logrocket.com/using-react-children-prop-with-typescript/).
-----------------------------------------------------------------------------------------------------------------------------------
## TLDR
```typescript
type ReactText = string | number;
type ReactChild = ReactElement | ReactText;

interface ReactNodeArray extends Array<ReactNode> {}
type ReactFragment = {} | ReactNodeArray;
type ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined;

type Props = {
  children: ReactNode
}

// source: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d076add9f29db350a19bd94c37b197729cc02f87/types/react/index.d.ts
```
or you can use `React.PropsWithChildren`.

File: packages/ui/components/Grid.tsx [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+  media?: Media;
+  className?: string;
+}
+type colSpan = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 'auto';
```
Comment (feodorrp) [inline packages/ui/components/Grid.tsx:12]: I highly recommend using Pascal Case for type definitions.
